### PR TITLE
Short branch

### PR DESCRIPTION
**Pull request by stephenRollinson04**
Fixing the hyperlinking.

### CODE REVIEW

**Review by stephenRollinson04 — COMMENTED** [review]
Changes are mostly due to adding background colour of the dog trainers page.

**Review by rsbennett22 — APPROVED** [review]

**Review by rsbennett22 — APPROVED** [review]

**Review by stephenRollinson04 — COMMENTED** [review]
No changes are nessesary.

**Review by rsbennett22 — APPROVED** [review]

File: GroupProject/dogWalkers/templates/dogWalkers/dogWalkersPage.html [hunk 1]
```diff
@@ -1,3 +1,272 @@
-#{% load static %}
```
Comment (stephenRollinson04) [inline GroupProject/dogWalkers/templates/dogWalkers/dogWalkersPage.html:1]: Irrelevent

File: GroupProject/dogWalkers/templates/dogWalkers/dogWalkersPage.html [hunk 2]
```diff
@@ -1,3 +1,272 @@
...
 <!DOCTYPE html>
-<h1>Congrats if you see this header!</h1>
```
Comment (stephenRollinson04) [inline GroupProject/dogWalkers/templates/dogWalkers/dogWalkersPage.html:3]: Irrelevent.

File: GroupProject/homepage/templates/homepage/home.html [hunk 3]
```diff
@@ -1,24 +1,42 @@
-{% load static %}
```
Comment (stephenRollinson04) [inline GroupProject/homepage/templates/homepage/home.html:1]: Irrelevent

**Pull request closed without merging**